foxtrot-core/src/main/java/com/flipkart/foxtrot/core/table/impl/TableMapStore.java

```diff
@@ -42,17 +42,18 @@
 import org.slf4j.LoggerFactory;
 
 import java.io.IOException;
+import java.io.Serializable;
 import java.util.Collection;
 import java.util.Map;
 import java.util.Properties;
 import java.util.Set;
 import java.util.concurrent.TimeUnit;
 
-public class TableMapStore implements MapStore<String, Table> {
+public class TableMapStore implements MapStore<String, Table>, Serializable {
     public static final String TABLE_META_INDEX = "table-meta";
     public static final String TABLE_META_TYPE = "table-meta";
     private static final Logger logger = LoggerFactory.getLogger(TableMapStore.class.getSimpleName());
-    private final ElasticsearchConnection elasticsearchConnection;
+    private static ElasticsearchConnection elasticsearchConnection;
```

Comment:
can keep this private final only and put an instance variable in `Factory implements MapStoreFactory`

Reply:
All variables defined in Factory needs to be serializable and since ElasticsearchConnection doesn't implement Serailizable, have removed this variable from there and made it static